components/library/DateRangePicker/tabs.ts

```diff
@@ -0,0 +1,255 @@
+import { Tab } from '@/types';
+
+const DateRangePicker = `import React, { useState, useRef, useEffect, useCallback } from 'react';
+import {
+  Popover,
+  Button,
+  TextField,
+  Box,
+  DatePicker,
+  Icon,
+  OptionList,
+  Scrollable,
+  InlineGrid,
+  BlockStack,
+  InlineStack
+} from '@shopify/polaris';
+import { ArrowRightIcon } from '@shopify/polaris-icons';
+
+export const DateRangePicker = ({ onDateRangeSelect }) => {
+  const today = new Date(new Date().setHours(0, 0, 0, 0));
+  const yesterday = new Date(
+    new Date(new Date().setDate(today.getDate() - 1)).setHours(0, 0, 0, 0)
+  );
+  const lastMonth = new Date(new Date().setMonth(today.getMonth() - 1));
```

Comment:
Looks like this line along with the `useRef` import aren't in the actual code file. I'm guessing you copied this over and then made additional changes to the other file, so just make sure to copy over the latest version here. I'm going to update the repo soon so it reads the source directly instead of needing to copy/paste as text so it'll be more straightforward in the future